cmd/cloud/workspace.go

```diff
@@ -71,6 +147,47 @@ func workspaceSwitch(cmd *cobra.Command, out io.Writer, args []string) error {
 	return workspace.Switch(id, astroClient, out)
 }
 
+func addWorkspaceUsers(cmd *cobra.Command, args []string, out io.Writer) error {
+	var email string
+
+	// if an email was provided in the args we use it
+	if len(args) > 0 {
+		email = args[0]
+	}
+
+	cmd.SilenceUsage = true
+	return user.AddWorkspaceUser(email, role, "", out, astroCoreClient)
+}
+
+func listWorkspaceUsers(cmd *cobra.Command, out io.Writer) error {
+	cmd.SilenceUsage = true
+	return user.ListWorkspaceUsers(out, astroCoreClient, "")
+}
+
+func updateWorkspaceUsers(cmd *cobra.Command, args []string, out io.Writer) error {
+	var email string
+
+	// if an email was provided in the args we use it
+	if len(args) > 0 {
+		email = args[0]
+	}
+
+	cmd.SilenceUsage = true
+	return user.UpdateWorkspaceUserRole(email, role, "", out, astroCoreClient)
+}
+
+func removeWorkspaceUsers(cmd *cobra.Command, args []string, out io.Writer) error {
+	var email string
+
+	// if an email was provided in the args we use it
+	if len(args) > 0 {
+		email = args[0]
+	}
+
+	cmd.SilenceUsage = true
+	return user.DeleteWorkspaceUser(email, "", out, astroCoreClient)
```

Comment:
Call this function RemoveWorkspaceUser, since we are not deleting them from the system, but just removing them from a particular workspace

Reply:
okay sounds good